### PR TITLE
Fix error message when session expires 

### LINKED ISSUE #7613: Modify the error message on the user session expired scenario
**Describe the bug**

When a session is getting expired, the notification error is thrown as "Authentication credentials were not provided"

**To Reproduce**

1. Stay of a page until you're session is expired
2. See error

**Expected behavior**

rephrase the notification to show as "Session expired. Please Login again"

**Screenshots**

image

### PR DESCRIPTION
**Pull request by AshrafMd-1**
## Proposed Changes

- Fixes #7613 
- Change 1
- Change 2
- More?

@coronasafe/care-fe-code-reviewers @coronasafe/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.coronasafe.network/coronasafe-care-documentation/architecture/architecture-and-layering-of-care).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

**Review by khavinshankar — APPROVED** [review]

File: src/Utils/Notifications.js [hunk 1]
```diff
@@ -70,6 +70,11 @@ export const Success = ({ msg }) => {
 
 /** Error message handler */
 export const Error = ({ msg }) => {
+  // Handle session expiry
+  if (msg === "Authentication credentials were not provided.") {
+    notify("Session expired. Please Login again.", "error");
+    return;
+  }
```
Comment (khavinshankar) [inline src/Utils/Notifications.js]: can you move to auth component or context, this seems to be odd place to have it

**Review by khavinshankar — CHANGES_REQUESTED** [review]

File: src/Utils/Notifications.js (hunk 1 above)
Comment (AshrafMd-1) [reply src/Utils/Notifications.js:77]: I mean i will check if there is any place 

But whenever there is a error notification its gonna display 'Authentication credentials were not provided.' so I thought keeping it in the notification.js is the best thing

**Comment by AshrafMd-1** [comment]
@khavinshankar can you review it once

cc: @nihal467

File: src/Utils/request/handleResponse.ts [hunk 2]
```diff
@@ -25,6 +25,15 @@ export default function handleResponse(
       navigate(`/session-expired?redirect=${window.location.href}`);
     }
 
+    // Handle session expiry
+    if (
+      error?.detail &&
+      error.detail === "Authentication credentials were not provided."
+    ) {
```
Comment (rithviknishad) [inline src/Utils/request/handleResponse.ts]:
```suggestion
    if (error?.detail === "Authentication credentials were not provided.") {
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]

**Review by rithviknishad — APPROVED** [review]

**Comment by nihal467** [comment]
LGTM

**Review by khavinshankar — APPROVED** [review]

**Pull request merged**